Updated the cloud portal article with the new dashboard (#4417)
umbraco/UmbracoDocs: Umbraco-Heartcore/Getting-Started-Cloud/The-Cloud-Portal/index.md

After you have logged into the [Umbraco Cloud Portal](https://www.umbraco.io), you are presented with the project list overview.

![Portal Project Overview](images/Heartcore-dashboard.png)

> From here you can see a list of all your projects. If you have several projects, you can sort specific projects in groups.

Author: ```suggestion
From here you can see a list of all your projects. If you have multiple projects, you can sort specific projects into groups.
```
